tests/data/user_data.1.txt

```diff
@@ -3,13 +3,3 @@ write_files:
 -   content: blah
     path: /etc/blah.ini
     permissions: 493
-
-system_info:
- package_mirrors:
-   - arches: [i386, amd64, blah]
-     failsafe:
-      primary: http://my.archive.mydomain.com/ubuntu
-      security: http://my.security.mydomain.com/ubuntu
-     search:
-      primary: []
-      security: []
```

Comment:
so this had to be removed because this is testing the unsupported behavior of setting `system_info` via user data? thus, it is now just baked into the cfg in the test below?